action.yml

```diff
@@ -0,0 +1,37 @@
+name: "Ruff"
+description: "An extremely fast Python linter."
+author: "Ruff Contributor(s)"
+inputs:
+  options:
+    description:
+      "Options passed to Ruff. Use `ruff --help` to see available options. 
+      Default: 'check'"
+    required: false
+    default: "check"
+  src:
+    description: "Source to run ruff. Default: '.'"
+    required: false
+    default: "."
```

Comment:
What's your opinion on using/allowing an array for src and `options`? For example, github uses an array for the arguments of a docker container:  https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsargs

Nit: Should we rename `options` to arguments?

Reply:
rename 'options' to 'arguments':  Doesn't matter to me.  They are pretty synonomous -- but also arguments can have the negative connotation of 'arguing', 'lawyers', etc...  :-) ...  again, as long as clear it doesn't matter.  

Array-v-String:  I believe it would be easier for those wanting to use an action to rely on the string that they would type in their local shell and put that in the `src` field.  Otherwise, they'll need to translate to alternate syntax.  Again, not necessarily a problem [ and actually somewhat cleaner ], but again, I'm thinking about this action as being something as easy as possible for people to be able to use.  I think this would be harder for people for src, and definitely harder for options/args [ there ordering is important, etc ].  But ...?